packages/react/src/components/hero/Hero.stories.tsx

```diff
@@ -0,0 +1,498 @@
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent />
+    </Hero>
+  );
+};
+
+export const PlaygroundForKoros = (args) => {
+  const heroProps: HeroProps = {
+    koros: {
+      type: args.type,
+      dense: !!args.dense,
+      hide: !!args.hide,
+      forcedDirection: args.forcedDirection || undefined,
+      ...args.koros,
+    },
+    theme: {
+      '--background-color': '#9fc9eb',
+      '--koros-color': args.color || '#9fc9eb',
+      ...args.theme,
+    },
+    imageSrc: imageFile,
+  };
+
+  return (
+    <Hero {...heroProps}>
+      <DefaultContent />
+    </Hero>
+  );
+};
+
+PlaygroundForKoros.argTypes = {
+  type: {
+    defaultValue: 'basic',
+    control: {
+      type: 'select',
+      options: ['basic', 'beat', 'pulse', 'storm', 'wave', 'calm'],
+    },
+  },
+  color: { control: { type: 'color' } },
+  hide: {
+    control: 'boolean',
+  },
+  dense: {
+    control: 'boolean',
+  },
+  flipHorizontal: {
+    defaultValue: true,
+    control: 'boolean',
+  },
+};
+
+export const EmbeddedToPage = (args) => {
+  const { preset, variant } = args;
+  const BasicImageVersion = () => {
+    return <ImageLeftOrRight variant={variant} />;
+  };
+  const BackgroundImageVersion = () => {
+    const theme = variant === 'backgroundImage' ? { '--bottom-koros-color': 'var(--color-fog)' } : {};
+    return <WithBackgroundImage variant={variant} theme={theme} />;
+  };
+  const NoImage = () => {
+    const heroPresets = ['blueAndGreen', 'whiteWithoutKoros', 'blackAndWhite'];
+    return <WithoutImage heroType={heroPresets[parseInt(preset, 10) - 1]} />;
+  };
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      <NavigationComponent />
+      {(variant === 'imageLeft' || variant === 'imageRight') && <BasicImageVersion />}
+      {(variant === 'backgroundImage' || variant === 'diagonalKoros') && <BackgroundImageVersion />}
+      {variant === 'noImage' && <NoImage />}
+      {variant === 'imageBottom' && <ImageBottom />}
+      <Section color="secondary">
+        <h1 className="heading-xl">Component after hero</h1>
+        This component shows padding after hero
+      </Section>
+    </div>
+  );
+};
+
+EmbeddedToPage.argTypes = {
+  koros: {
+    description: '*** Koros is not passed the the components ***',
+    control: false,
+  },
+  theme: {
+    description: '*** Theme is not passed the the components ***',
+    control: false,
+  },
+  variant: {
+    defaultValue: 'noImage',
+    control: {
+      type: 'select',
+      options: ['imageLeft', 'imageRight', 'backgroundImage', 'imageBottom', 'diagonalKoros', 'noImage'],
+    },
+  },
+  preset: {
+    defaultValue: '1',
+    control: {
+      type: 'select',
+      options: ['1', '2', '3'],
+    },
+    table: {
+      type: {
+        summary:
+          'Changes to another preset of the selected variant. Storybook control, not an actual component property',
+      },
+    },
+  },
+};
+
+export const PlaygroundForDiagonalKoros = (args) => (
```

Comment:
Do we need a loki test for broken hero?
What is the meaning of the broken hero component?

Reply:
Koros heights are exported with `getShapeHeight` in Koros.tsx.

I can make this Hero specific feature and and remove it from Koros.

Reply:
Removed the intentionally broken version. Removed  changes in Koros component, except getShapeHeight